examples/simple-web-proof/vlayer/src/components/organisms/MintStep/Container.tsx

```diff
@@ -29,6 +30,16 @@ export const MintStep = () => {
     hash: txHash,
   });
 
+  const { chain, error: syncChainError } = useSyncChain(
+    import.meta.env.VITE_CHAIN_NAME,
+  );
```

Comment:
Is that required here given that we already check it in `useTwitterAccountProof`?

Reply:
yes user could change something in wallet.